lucetc/lucetc/main.rs

```diff
@@ -72,7 +72,18 @@ pub fn run(opts: &Options) -> Result<(), Error> {
         _ => Err(format_err!("provided too many inputs: {:?}", opts.input)),
     }?;
 
+    let mut validator = if !opts.witx_specs.is_empty() {
+        Some(Validator::load(&opts.witx_specs)?.with_wasi_exe(opts.wasi_exe))
+    } else {
+        None
+    };
+
     let mut bindings = Bindings::empty();
+    if opts.wiggle_bindings {
+        if let Some(ref v) = validator {
+            bindings.extend(&lucet_wiggle_generate::bindings(v.doc()))?;
+        }
+    }
```

Comment:
Nice :slightly_smiling_face: Very cool to see how concise this ends up being, all the way up here in `lucetc/main.rs`